### ISSUE TITLE
Does tfa.metrics.F1Score calculate f1 score on each batch (or) on full eval set?

### ISSUE THREAD

**Issue by John-8704** [tensorflow/addons#1875]
Please update the documentation to include the information regarding whether the metric calculates f1 per each batch and then averages it out (or) does it calculate on full validation set?

**Comment by facaiy** [comment]
Hi, John. F1Score calculates on the whole data which it had seen before.

**Comment by facaiy** [comment]
I'll close the issue, and feel free to reopen it :-)
